src/settings.cpp

```diff
@@ -981,6 +985,36 @@ static bool DecodeHexText(const char *pos, uint8_t *dest, size_t dest_size)
 	return *pos == '|';
 }
 
+/**
+ * Load BaseGraphics set selection and configuration.
+ */
+static void GraphicsSetLoadConfig(IniFile &ini)
+{
+	if (IniGroup *group = ini.GetGroup("misc"); group != nullptr) {
+		/* Load old setting first. */
+		if (IniItem *item = group->GetItem("graphicsset"); item != nullptr && item->value) BaseGraphics::ini_data.name = *item->value;
```

Comment:
Interesting syntax inside if that I've not seen before.

Reply:
https://youtu.be/tsG95Y-C14k?t=114

Reply:
https://en.cppreference.com/w/cpp/language/if#If_statements_with_initializer